packages/lix-sdk/src/query-utilities/get-lowest-common-ancestor-v2.test.ts

```diff
@@ -0,0 +1,177 @@
+import { test, expect } from "vitest";
+import { openLixInMemory } from "../open/openLixInMemory.js";
+import { newLixFile } from "../newLix.js";
+import type { Change, ChangeGraphEdge, NewChange } from "../database/schema.js";
+import { getLowestCommonAncestorV2 } from "./get-lowest-common-ancestor-v2.js";
+
+test("it should find the common parent of two changes recursively", async () => {
+	const lix = await openLixInMemory({
+		blob: await newLixFile(),
+	});
+
+	const mockChanges = [
```

Comment:
This test is not sharp enough a call to "getRootChange" would be sufficient in this case as well. I would at least add some more history before the common to ensure the query stops as expected?